FirebaseAuth/Sources/Utilities/FIRAuthWebUtils.m

```diff
@@ -107,19 +107,29 @@ + (void)fetchAuthDomainWithRequestConfiguration:(FIRAuthRequestConfiguration *)r
                 // The sequence of supportedAuthDomains matters. ("firebaseapp.com", "web.app")
```

Comment:
Perhaps look into either changing the comment or the moved code while this bit is being refactored?

Comment is still wrong: the order of supportedAuthDomains doesn't matter as it's being iterated in the inner for loop rather than the outer.